internal/nodes/avi_model_tls_passthrough.go

```diff
@@ -66,6 +66,10 @@ func (o *AviObjectGraph) BuildVSForPassthrough(vsName, namespace, hostname, key
 		VipNetworks: lib.GetVipNetworkList(),
 	}
 
+	if lib.GetT1LRPath() != "" {
```

Comment:
@monotosh-avi : At line number 53, We need to set "vrf-context" of passthrough VS node to empty for NSX-T environment.

Reply:
good catch. updated.